static/src/javascripts/projects/common/modules/commercial/membership-engagement-banner-parameters.js

```diff
@@ -1,32 +1,30 @@
 // @flow
 import config from 'lib/config';
+import reportError from 'lib/report-error';
 import {
+    getSync as getGeoLocation,
     getSupporterCountryGroup,
     extendedCurrencySymbol,
 } from 'lib/geolocation';
 import { supportContributeURL } from './support-utilities';
+import { getBannerGoogleDoc } from './contributions-google-docs';
 
-const baseParams = {
-    minArticles: 3,
-    campaignCode: 'gdnwb_copts_memco_banner',
-};
-
-const engagementBannerCopy: string = `<strong>The Guardian is editorially independent &ndash;
+const engagementBannerControl: string = `<strong>The Guardian is editorially independent &ndash;
     our journalism is free from the influence of billionaire owners or politicians.
     No one edits our editor. No one steers our opinion.</strong> And unlike many others, we haven’t put
     up a paywall as we want to keep our journalism open and accessible. But the revenue we get from
     advertising is falling, so we increasingly need our readers to fund our independent, investigative reporting.`;
 
 const initialContribution = 1;
 
-const supporterCost = (location: string): string => {
+const supporterCost = (location: string, askAmount: number): string => {
     const countryGroup = getSupporterCountryGroup(location);
+    const amount = askAmount || initialContribution;
```

Comment:
we have default parameters in ES6 so we could just set this there? and not pass into `supporterEngagementCtaCopyControl`. Feels a bit weird to pass in the default value while also having it as a fallback

Reply:
especially because the name of the variable is quite misleading (maybe leftover from some test)

Reply:
...by name of variable, I mean `initialContribution`

Reply:
done